tar/multitape/multitape_write.c

```diff
@@ -104,11 +104,11 @@ static int flushtape(TAPE_W *, int);
 /* Initialize stream. */
 static int
 stream_init(struct stream * S, chunkify_callback callback, void * cookie,
-    int no_chunkifier)
+    int use_chunkifier)
 {
 
 	/* Create chunkifier. */
-	if (!no_chunkifier) {
+	if (use_chunkifier) {
```

Comment:
When I looked at the code again, I really didn't like the `(!no_chunkifier)` part.